### ISSUE TITLE
Spotify only load black screen

### ISSUE THREAD

**Issue by danrynr** [catppuccin/spicetify#14]
I manage applying the theme without any errors. But when i open spotify this black screen showup.
![image](https://user-images.githubusercontent.com/29006513/164974184-adc2c0c3-fef5-4e06-8880-3ec620d14a12.png)

Spotify version : 1.1.72.439.gc253025e
Spicetify version : 2.9.6

**Comment by danrynr** [comment]
FIXED! 👍🏻
disable "spicetify-marketplace"
set check for updates to "1"

**Comment by davidbgonz** [comment]
I've been having the same blank screen on spotify 1.1.80 and spicetify 2.9.7. There are a number of issues that I have been tracking on https://github.com/spicetify/spicetify-cli/issues. It looks like once we get 1.1.83 the issues will be resolved.
The steps I did to get thinks working was installing 1.1.83. For arch I had to get it from the spotify-edge package from AUR .

1. Uninstall current spotify
2. Install latest spotify
3. `spicetify backup apply`
4. Profit
